nixos/modules/services/networking/thelounge.nix

```diff
@@ -50,15 +61,24 @@ in {
         Documentation: <link xlink:href="https://thelounge.chat/docs/server/configuration" />
       '';
     };
+
+    plugins = mkOption {
+      default = [ ];
+      type = types.listOf types.package;
+      example = literalExpression "[ pkgs.nodePackages.thelounge-theme-solarized ]";
+      description = ''
+        The Lounge plugins to install.
```

Comment:
It would be awesome if you would give a hint, how to discover these plugins in nixpkgs.